### ISSUE TITLE
Difference between v2.1 and v2.2

### ISSUE THREAD

**Issue by asychev** [bigtreetech/Manta-M4P#25]
Are there any significant difference between current v2.2 version and previous v2.1?
In `Hardware` folder only 2.1 documentation available, can it be updated?

**Comment by 1954Doc** [comment]
Same here.

Or. just tell us the technical information (pin ID's, signals, etc.) are the same so we can get on with debugging.

**Comment by warpwookie** [comment]
Agreed, the Chip ID is different on the V2.2 as well I've been trying to setup my V2.2 for about 3 days now and not having great luck. The Chip on my board is a G0B0 not a G0B1. could we get some support here???

**Comment by warpwookie** [comment]
I just flashed a new board because Mainsail said my klipper was out of date on the version that came out of the box version 10 something of klipper. So I flashed it using the G0B1 config and I cannot see it in lsusb or ls /dev/serial/by-id or by-path/ Help! the version of klipper I'm working with on my host is v0.12.0-128

**Comment by satbe** [comment]
Although v2.2 has the B0 STM32 Chip, the HW seems to be compatible with V2.1. I had to replace an M4P because of a dead analog input. The original version was 2.1, the new one was 2.2.  I built the binary acording their manual (chose 
G0B1). I also updated Klipper before I built the binary. A full HW power cycle was necessary to update the M4P from flash. A Linux system restart was not enough. After that I got a new usb ID which I added to printer.cfg. My main mistake preventing the board to update from flash was to not HW power cycle.
